src/Components/ServiceScene/LogsListScene.tsx

```diff
@@ -26,20 +26,22 @@ import {
 import { logger } from '../../services/logger';
 import { Options } from '@grafana/schema/dist/esm/raw/composable/logs/panelcfg/x/LogsPanelCfg_types.gen';
 import { narrowLogsVisualizationType, narrowSelectedTableRow, unknownToStrings } from '../../services/narrowing';
+import { LogLineState } from '../Table/Context/TableColumnsContext';
 
 export interface LogsListSceneState extends SceneObjectState {
   loading?: boolean;
   panel?: SceneFlexLayout;
   visualizationType: LogsVisualizationType;
   urlColumns?: string[];
+  urlLogLineState?: LogLineState;
   selectedLine?: SelectedTableRow;
   $timeRange?: SceneTimeRangeLike;
   displayedFields: string[];
 }
 
 export class LogsListScene extends SceneObjectBase<LogsListSceneState> {
   protected _urlSync = new SceneObjectUrlSyncConfig(this, {
-    keys: ['urlColumns', 'selectedLine', 'visualizationType', 'displayedFields'],
+    keys: ['urlColumns', 'selectedLine', 'visualizationType', 'displayedFields', 'urlLogLineState'],
```

Comment:
```suggestion
    keys: ['urlColumns', 'selectedLine', 'visualizationType', 'displayedFields', 'tableLogLineState'],
```

I see you're also prefixing columns with url, but seems like a superfluous prefix, as all of them are "url" (urlSelectedLine, urlVisualizationType, etc).